library/src/main/java/com/github/runly/circularprogressview/CircularProgressDrawable.java

```diff
@@ -199,7 +200,31 @@ private void drawIndeterminate(Canvas canvas) {
 
         canvas.drawArc(mRect, mStartAngle, mSweepAngle, false, mPaint);
       }
-    } else if (mRunState != RUN_STATE_STOPPED) {
+    } else if (isPaused()) {
```

Comment:
I don't think so. `isPaused()` checks if `mRunState == RUN_STATE_PAUSED;` while `isStopped()` checks `mRunState == RUN_STATE_STOPPED;` !isStopped() could be that it is paused, running, stopping, or other states.